device-types/HPE/Aruba-2920-48G.yaml

```diff
@@ -5,11 +5,13 @@ slug: hpe-aruba-2920-48g
 part_number: J9728A
 u_height: 1
 is_full_depth: false
+weight: 5420
+weight_unit: g
 power-ports:
   - name: PS1
     type: iec-60320-c14
-    maximum_draw: 165
-    allocated_draw: 70
+    maximum_draw: 70
+    allocated_draw: 27
```

Comment:
It appears that the PSU is hot swappable. Thus it should be modelled under `module-bays`, and then the associated PSU module should be modelled in the module-types folder if they do not already exist there

If I am wrong about this please let me know and mark this as resolved